Resources/input/fsh/extensions/GEM_ERP_EX_LastMedicationDispense.fsh

```diff
@@ -0,0 +1,12 @@
+Extension: GEM_ERP_EX_LastMedicationDispense
+Id: GEM-ERP-EX-LastMedicationDispense
+Title: "Last Medication Dispense"
+Description: "This extension should be used in the Task ressource. It shows the timestamp of the last dispense."
+* ^url = "https://gematik.de/fhir/erp/StructureDefinition/GEM_ERP_EX_LastMedicationDispense"
+* ^version = "1.3.0"
+* ^status = #draft
+* . ^short = "Timestamp which is set when a medication for this task has been dispensed."
+* . ^definition = "This field allows the frontend to display to the user that a medication has been dispensed before it was closed by the pharmacy."
```

Comment:
Lass uns nicht von "frontend" sondern vom FdV reden, um die Begrifflichkeiten nicht aufzuweichen.

Reply:
changed to E-Rezept-FdV